### DISCUSSION TITLE
Suddenly can not build with LGFX

### DISCUSSION THREAD

**Discussion by dekesone** [lovyan03/LovyanGFX#598] [General]
For some reason I am suddenly unable to successfully compile any projects that use LGFX.  I have tried numerous solutions with no success.  I've removed and re-added the libraries (manually as well as using the libraries manager tool).  

I finally got to the point where I just create a blank new project, add the library reference to LGFX 1.1.16, and add the `#include <LovyanGFX.hpp>` to the main.cpp and it immediate fails to build.  There's no code written whatsoever other than adding the include reference and the main.cpp is completely empty.  Removing the `#include <LovyanGFX.hpp` does allow the build to successfully complete.

I'm developing in VS Code using PlatformIO. All of the correct LGFX files exist in my .pio/libdeps folder 

 Here is the error I receive on every build:  Any suggestions on how to resolve this issue?

CONFIGURATION: https://docs.platformio.org/page/boards/espressif32/esp32-s3-devkitc-1.html
PLATFORM: Espressif 32 (6.7.0) > Espressif ESP32-S3-DevKitC-1-N8 (8 MB QD, No PSRAM)
HARDWARE: ESP32S3 240MHz, 320KB RAM, 8MB Flash
DEBUG: Current (esp-builtin) On-board (esp-builtin) External (cmsis-dap, esp-bridge, esp-prog, iot-bus-jtag, jlink, minimodule, olimex-arm-usb-ocd, olimex-arm-usb-ocd-h, olimex-arm-usb-tiny-h, olimex-jtag-tiny, tumpa)
PACKAGES:
 - framework-arduinoespressif32 @ 2.0.17+sha.5e19e08
 - tool-esptoolpy @ 1.40501.0 (4.5.1) 
 - toolchain-riscv32-esp @ 8.4.0+2021r2-patch5
 - toolchain-xtensa-esp32s3 @ 8.4.0+2021r2-patch5
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
LDF Modes: Finder ~ chain, Compatibility ~ soft
Found 34 compatible libraries
Scanning dependencies...
Dependency Graph
|-- LovyanGFX @ 1.1.16
Building in release mode
Compiling .pio\build\esp32-s3-devkitc-1\lib440\LovyanGFX\lgfx\Fonts\IPA\lgfx_font_japan.c.o
xtensa-esp32s3-elf-gcc: error: CreateProcess: No such file or directory
Compiling .pio\build\esp32-s3-devkitc-1\lib440\LovyanGFX\lgfx\Fonts\efont\lgfx_efont_cn.c.o
*** [.pio\build\esp32-s3-devkitc-1\lib440\LovyanGFX\lgfx\Fonts\IPA\lgfx_font_japan.c.o] Error 1
xtensa-esp32s3-elf-gcc: error: CreateProcess: No such file or directory
*** [.pio\build\esp32-s3-devkitc-1\lib440\LovyanGFX\lgfx\Fonts\efont\lgfx_efont_cn.c.o] Error 1

**Comment by tobozo** [comment]
hi,

> `Error 1 xtensa-esp32s3-elf-gcc: error: CreateProcess: No such file or directory`

this is either the antivirus blocking access to the executable or a corrupt platformio toolchain

closing this as not a LovyanGFX issue
